README.md

```diff
@@ -356,89 +423,431 @@ tondev tonos-cli update
+TONDev Version: 0.5.0
+Use: tondev networks add name endpoints [options]
+Args:
+    name
+    endpoints  Comma separated endpoints
+Options:
+    --help, -h   Show command usage
+    --force, -f  Overwrite key if already exists
+```
+
+#### List registered networks
+
+This command lists all registered networks and their public endpoints.
+
+```bash
+tondev networks list
+```
+
+Result:
 
-Prints the network list.
+```bash
+$ tondev networks list
+
+Network         Endpoints
+--------------  ------------
+main (Default)  main.ton.dev
+dev             net.ton.dev
+```
 
-#### Default Command
+#### Set default network
 
-Sets the specified network as default.
+This command sets a previously added network as default (initially the mainnet is used by default).
 
+```bash
+tondev signers default network_name
+```
+
+#### Delete a network
+
+This command deletes a network from tondev registry.
+
+```bash
+tondev networks delete network_name
+```
 ### Contract Management
 
-Contract management of the tondev give you an ability to easily deploy and run 
-your smart contracts on a blockchain network(s).
+Contract management in tondev gives you the ability to easily deploy and run 
+your smart contracts on blockchain network(s).
+
+#### View contract info
+
+This command displays a summary (selected network and signer, account address and account status). Contract ABI and TVC files are required to run it.
+
+```bash
+tondev contract info abi_filename
+```
+
+Result example:
+
+```bash
+$ tondev contract info Contract
+
+Configuration
+
+  Network: main
+  Signer:  sign1
+
+Address: 0:0435cb4e70585759ac514bb9fd1770caeb8c3941d882b5a16d589b368cb49261
+Account: Not exists
+```
+
+Network, signer and account address parameters can be overridden with the following options:
+
+```bash
+$ tondev contract info -h
+TONDev Version: 0.5.0
+Use: tondev contract info file [options]
+Args:
+    file  ABI file
+Options:
+    --help, -h     Show command usage
+    --network, -n  Network name
+    --signer, -s   Signer key name
+    --address, -a  Account address
+```
+
+#### Deploy contract
 
-#### Info Command
+This command deploys a contract to the blockchain. Contract ABI and TVC files are required to run it.
 
-Show account information.
+```bash
+tondev contract deploy abi_filename
+```
+
+Command displays deployment summary and requests constructor function parameters. Result example:
+
+```bash
+$ tondev contract deploy Contract
+
+Configuration
+
+  Network: dev
+  Signer:  sign1
+
+Address: 0:0435cb4e70585759ac514bb9fd1770caeb8c3941d882b5a16d589b368cb49261
+
+Enter constructor parameters
+
+  param1 (uint256[]): value
+
+Enter constructor parameters
+
+  param2 (uint8): value
+
+Deploying...
+```
+
+Network, signer and account address parameters can be overridden and constructor parameters specified in the deploy command with the following options:
+
+```bash
+$ tondev contract deploy -h
+TONDev Version: 0.5.0
+Use: tondev contract deploy file function [options]
```

Comment:
Add a sample of multisig deploy with many custodians in one line - we need to show that tondev does not require any {} symbols and that it is easier than tonos-cli console input